sig-release/cherry-picks.md

```diff
@@ -0,0 +1,73 @@
+# Overview
```

Comment:
My understanding was that these files will move from k/community/devel to k/community/devel/sig-release/cherry-picks.md. :thinking:

Reply:
So, it is better to create folders in accordance with the [classification](https://github.com/kubernetes/community/issues/3097) of files by SIGs, right?

Reply:
Yes, just like how design proposals are grouped by SIG: https://github.com/kubernetes/community/tree/master/contributors/design-proposals